app/models/solidus_paypal_braintree/source.rb

```diff
@@ -1,12 +1,13 @@
 module SolidusPaypalBraintree
-  class Source < ApplicationRecord
+  class Source < Spree::PaymentSource
     PAYPAL = "PayPalAccount"
     APPLE_PAY = "ApplePayCard"
     CREDIT_CARD = "CreditCard"
 
     belongs_to :user, class_name: Spree::UserClassHandle.new
     belongs_to :payment_method, class_name: 'Spree::PaymentMethod'
     has_many :payments, as: :source, class_name: "Spree::Payment"
+    has_many :wallet_payment_sources, class_name: 'Spree::WalletPaymentSource', as: :payment_source, inverse_of: :payment_source
```

Comment:
You shouldn't need this if you inherit from `SolidusSupport.payment_source_parent_class`